src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -586,6 +586,49 @@ class LagPortSpec extends HealthCheckSpecification {
         lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
     }
 
+    @Tidy
+    def "Unable decrease bandwidth on LAG port lower than connected flows bandwidth sum"() {
+        given: "Flows on a LAG port with switch ports"
+        def switchPair = topologyHelper.getSwitchPairs().first()
+        def testPorts = topology.getAllowedPortsForSwitch(switchPair.src).takeRight(2).sort()
+        assert testPorts.size > 1
+        def maximumBandwidth = {
```

Comment:
`def maximumBandwidth = testPorts.sum { northbound.getPort(switchPair.src.dpId, it).maxSpeed }`